src/Microsoft.IdentityModel.Tokens/ECDsaSecurityKey.cs

```diff
@@ -103,22 +103,7 @@ public override int KeySize
         /// <returns>true if this supports the algorithm; otherwise, false.</returns>
         public override bool IsSupportedAlgorithm(string algorithm)
```

Comment:
I agree, we will remove this method and have users call: SecurityKey.CryptoProviderFactory.IsAlgorithmSupported(algorithm, this);